[2024-12-13T13:46:58.635+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-13T13:46:58.657+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_test.etl_dim_user manual__2024-12-13T13:46:56.368964+00:00 [queued]>
[2024-12-13T13:46:58.670+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_test.etl_dim_user manual__2024-12-13T13:46:56.368964+00:00 [queued]>
[2024-12-13T13:46:58.671+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-13T13:46:58.694+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_user> on 2024-12-13 13:46:56.368964+00:00
[2024-12-13T13:46:58.708+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2201) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-13T13:46:58.712+0000] {standard_task_runner.py:72} INFO - Started process 2206 to run task
[2024-12-13T13:46:58.711+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_test', 'etl_dim_user', 'manual__2024-12-13T13:46:56.368964+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/test_etl_dag.py', '--cfg-path', '/tmp/tmp7w9r74og']
[2024-12-13T13:46:58.714+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask etl_dim_user
[2024-12-13T13:46:58.785+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_test.etl_dim_user manual__2024-12-13T13:46:56.368964+00:00 [running]> on host c71f4d197b13
[2024-12-13T13:46:58.898+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_test' AIRFLOW_CTX_TASK_ID='etl_dim_user' AIRFLOW_CTX_EXECUTION_DATE='2024-12-13T13:46:56.368964+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-13T13:46:56.368964+00:00'
[2024-12-13T13:46:58.900+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-13T13:46:58.936+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-13T13:52:44.853+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-13T13:52:45.618+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-13T13:52:45.637+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_test, task_id=etl_dim_user, run_id=manual__2024-12-13T13:46:56.368964+00:00, execution_date=20241213T134656, start_date=20241213T134658, end_date=20241213T135245
[2024-12-13T13:52:46.000+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-13T13:52:46.639+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-13T13:52:46.644+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
